projects/ngrx.io/content/guide/router-store/index.md

```diff
@@ -28,3 +28,5 @@ import { AppComponent } from './app.component';
 })
 export class AppModule {}
 </code-example>
+
+> :warning: **@ngrx/router-store does not compatible with `ngrx-store-freeze`. Using together may cause unexpected behaviours**
```

Comment:
ngrx-store-freeze is deprecated in favor of the built in runtime checks, see our [migration guide](https://ngrx.io/guide/migration/v8#deprecation-of-ngrx-store-freeze) for more info.
The unexpected behaviours shouldn't occur with the built in checks.

Reply:
@timdeschryver  we are using 8.6 and it is not compatible with `strictStateImmutability:` either. Gives same result. It really took me a while figure that out. I think it would help many others if we put a little note to warn them.

Reply:
Depends on the configuration, the angular version, and what you add to the payload of the actions.
This is covered at https://ngrx.io/guide/store/configuration/runtime-checks#configuring-runtime-checks but perhaps it needs more info? 
I think the warning here should just point to the runtime checks.

Reply:
In fact, it's already added here https://ngrx.io/guide/router-store/configuration#default-router-state-serializer.

Reply:
@timdeschryver at least can we add some hint to imply this conflict could blow up root reducers. it is really time consuming to find out all these related together

Reply:
@chrstnbrn I'm not saying we shouldn't add this to the docs, but I think we should reference our docs from here. This way, all of the information remains at one location, and isn't shattered across the docs.